src/frontend/src/lib/services/address.services.ts

```diff
@@ -39,19 +55,51 @@ export const loadAddress = async (): Promise<{ success: boolean }> => {
 	return { success: true };
 };
 
-const saveEthAddressForFutureSignIn = async ({
+const loadBtcAddress = async ({
+	tokenId,
+	network
+}: {
+	tokenId: TokenId;
+	network: BitcoinNetwork;
+}): Promise<{ success: boolean }> =>
+	loadTokenAddress<BtcAddress>({
+		tokenId,
+		getAddress: (identity) =>
+			getBtcAddress({
+				identity,
+				network: network === 'mainnet' ? { mainnet: null } : { testnet: null }
```

Comment:
What's more important, loading mainnet or testnet? If mainnet, should we invert the if? i.e. default fallback

Note: Strictly a question, no preferences from my side

Reply:
I think I understand your point. That in any case, the default fallback should be mainnet. Make sense, changed